docs/static/docker.asciidoc

```diff
@@ -39,6 +39,45 @@ https://www.docker.elastic.co[www.docker.elastic.co].
 
 endif::[]
 
+==== Verifying the image
+
+Although it's optional, we highly recommend verifying the signatures included with your downloaded Docker images to ensure that the images are valid.
+
+Elastic images are signed with https://docs.sigstore.dev/cosign/overview/[Cosign] which is part of the https://www.sigstore.dev/[Sigstore] project. Cosign supports container signing, verification, and storage in an OCI registry. Install the appropriate https://docs.sigstore.dev/cosign/installation/[Cosign application]
+for your operating system.
+
+Run the following commands to verify the *elastic-agent* container image signature for {ls} v{version}:
```

Comment:
```suggestion
Run the following commands to verify the *logstash* container image signature for {ls} v{version}:
```

Reply:
I adjusted the suggestion just to remove the container image name altogether rather than have "Logstash" appear twice in the sentence.